src/main/java/com/piseth/java/school/schoolManagement/service/MonthlyScoreService.java

```diff
@@ -7,9 +7,12 @@
 import com.piseth.java.school.schoolManagement.model.MonthlyScore;
 
 public interface MonthlyScoreService {
+	
 	void addMonthlyScore(MonthlyScore monthlyScore);
 	
 	List<MonthlyScore> getMonthlyScore(Map<String, String> params);
 	
 	List<RankDTO> getRankDTO(Map<String, String> params);
+	
+	Map<String,Double> getListScoreByStudentIDYearMonth(Map<String, String> params);
```

Comment:
Since parameter of method is a map , it is very flexible. So the method name should be more general. 
You can consider method name like this : "listScore".